packages/earth-admin/src/pages-client/homepage.tsx

```diff
@@ -18,20 +18,56 @@
 */
 
 import * as React from 'react';
-import { useEffect } from 'react';
-import { navigate } from 'gatsby';
 
+import { useRequest } from 'utils/hooks';
 import { useAuth0 } from 'auth/auth0';
+import { AuthzGuards } from 'auth/permissions';
+import { getOrganizationStats } from 'services/organizations';
 
-/**
- * Just redirect to the default selectedGroup. (set in auth0.tsx)
- */
-const Homepage = () => {
+import { APP_NAME } from '../theme';
+import { ContentLayout, SidebarLayout } from 'layouts';
+import { Card, ErrorMessages } from 'components';
+
+const Homepage = (props) => {
   const { selectedGroup } = useAuth0();
-  useEffect(() => {
-    selectedGroup && navigate(`/${selectedGroup}`, { replace: true });
-  }, [selectedGroup]);
-  return <div>This is homepage - should be redirected to /:org</div>
-}
+  const { isLoading, data: organization, errors } = useRequest(() => getOrganizationStats(selectedGroup), {
+    permissions: AuthzGuards.accessOrganizationsGuard,
+    query: selectedGroup // when this changes, we refetch
+  });
+
+  return (
+    <>
+      <SidebarLayout isLoading={isLoading}>
+        <Card className="ng-margin-top" style={{ 'overflowY': 'scroll' }}>
+          {organization.name && (
+            <>
+              <h2 className="ng-text-display-m ng-margin-bottom-remove">{organization.name}</h2>
+              <p className="ng-margin-bottom-large">{organization.description}</p>
+              <p className="ng-margin-vertical"><strong>Slug: </strong>{organization.slug}</p>
+
+              <hr className="ng-hr-small" style={{ opacity: .3 }} />
```

Comment:
don't think this opacity results in the correct color from design, create a classname for it